mne_bids_pipeline/_main.py

```diff
@@ -32,46 +34,48 @@ def main():
         filename extension, separated by a '/'. For example, to run ICA, you
         would pass 'sensor/run_ica`. If unspecified, will run all processing
         steps. Can also be a tuple of steps."""))
-    parser.add_option(
+    parser.add_argument(
         '--root-dir', dest='root_dir', default=None,
         help="BIDS root directory of the data to process.")
-    parser.add_option(
+    parser.add_argument(
         '--subject', dest='subject', default=None,
         help="The subject to process.")
-    parser.add_option(
+    parser.add_argument(
         '--session', dest='session', default=None,
         help="The session to process.")
-    parser.add_option(
+    parser.add_argument(
         '--task', dest='task', default=None,
         help="The task to process.")
-    parser.add_option(
+    parser.add_argument(
         '--run', dest='run', default=None,
         help="The run to process.")
-    parser.add_option(
-        '--n_jobs', dest='n_jobs', type='int', default=None,
+    parser.add_argument(
+        '--n_jobs', dest='n_jobs', type=int, default=None,
         help="The number of parallel processes to execute.")
-    parser.add_option(
+    parser.add_argument(
         '--interactive', dest='interactive', action='store_true',
         help="Enable interactive mode.")
-    parser.add_option(
+    parser.add_argument(
         '--debug', dest='debug', action='store_true',
         help="Enable debugging on error.")
-    parser.add_option(
+    parser.add_argument(
         '--no-cache', dest='no_cache', action='store_true',
         help='Disable caching of intermediate results.')
-    options, args = parser.parse_args()
+    options = parser.parse_args()
     config = options.config
-    bad_msg = (
-        'You must specify a configuration file as a single argument '
-        'or with --config.'
-    )
+    config_switch = options.config_switch
+    bad = False
     if config is None:
-        if len(args) == 1:
-            config = args[0]
+        if config_switch is None:
+            bad = 'neither was provided'
         else:
-            raise ValueError(bad_msg)
-    elif len(args):
-        raise ValueError(bad_msg)
+            config = config_switch
+    elif config_switch is not None:
+        bad = 'both were provided'
+    if bad:
+        parser.error(
+            '😵 You must specify a configuration file either as a single '
```

Comment:
Hehe, I love this emoji but I realized in my terminal it's sometimes a little small and I cannot really make out the eyes :( So in #653 I switched from that emoji to a fat red X:
```suggestion
            '❌ You must specify a configuration file either as a single '
```